Tests/HarmonyTests/GenericNetworkDataSourceTests.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  GenericNetworkDataSourceTests.swift
+//  HarmonyTesting-iOS
+//
+//  Created by Kerim Sari on 22.09.2022.
+//
+
+import Foundation
+import XCTest
+import Nimble
+import Harmony
+import Alamofire
+
+@available(iOS 13.0, *)
+class GenericNetworkDataSourceTests: XCTestCase {
+    
+    private struct MockEntity: Decodable, Equatable {}
+    
+    func test_query_not_supported() {
```

Comment:
Just a suggestion. I am not sure if we have a convention or whether we, but we could make the name a bit more descriptive. 
- test_all_objects_query_is_not_supported or test_non_supported_queries_throw_error

Reply:
Done